build-js.sh

```diff
@@ -24,7 +24,7 @@ if [ "$1" == "-h" ] || [ "$1" == "--help" ] || [ "$1" == "-help" ]; then
   exit 1
 fi
 
-if [ -z $EMSCRIPTEN ]; then
+if [ -z "$EMSCRIPTEN" ]; then
```

Comment:
Quoting here prevents issues when running bash in Windows, where paths might contain white spaces.